test/test_other.py

```diff
@@ -3109,6 +3109,24 @@ def test_file_packager_embed(self):
     output = self.run_js('a.out.js')
     self.assertContained('hello data', output)
 
+  def test_file_packager_depfile(self):
+    create_file('data1.txt', 'data1')
+    ensure_dir('subdir')
+    create_file('subdir/data2.txt', 'data2')
+
+    self.run_process([FILE_PACKAGER, 'test.data', '--js-output=test.js', '--depfile=test.data.d', '--from-emcc', '--preload', '.'])
+    lines = read_file('test.data.d').split('\n')
+    split = lines.index(': \\')
+    before, after = set(lines[:split]), set(lines[split + 1:])
+    # Set comparison used because depfile is not order-sensitive.
+    self.assertTrue('test.data \\' in before)
+    self.assertTrue('test.js \\' in before)
+    self.assertTrue(FILE_PACKAGER + '.py \\' in after)
+    self.assertTrue('. \\' in after)
+    self.assertTrue('./data1.txt \\' in after)
+    self.assertTrue('./subdir \\' in after)
+    self.assertTrue('./subdir/data2.txt \\' in after)
```

Comment:
I wonder if it would be clearer to simply check the entire file contents. eg.

```
expected = '''
blah
blah
'''
   self.assertEqual(read_file('test.data.d'), expected)
```

Reply:
Fair enough.. this lgtm as is then.